qt/scientific_interfaces/ISISReflectometry/GUI/MainWindow/QtMainWindowView.cpp

```diff
@@ -102,10 +109,15 @@ void QtMainWindowView::initLayout() {
 
   // Create the presenter
   auto slitCalculator = std::make_unique<SlitCalculator>(this);
+  m_optionsDialogView = std::make_unique<QtOptionsDialogView>(this);
+  auto optionsDialogView = m_optionsDialogView.get();
+  auto optionsDialogModel = std::make_unique<OptionsDialogModel>();
+  auto optionsDialogPresenter = std::make_unique<OptionsDialogPresenter>(
```

Comment:
Might be a bit cleaner to do this to avoid a couple of local variables which are only used once:
```
auto optionsDialogPresenter = std::make_unique<OptionsDialogPresenter>(
    m_optionsDialogView.get(), std::make_unique<OptionsDialogModel>())
```

Reply:
Agreed, forgot to change this after testing it worked.